app/models/post.js

```diff
@@ -74,16 +79,14 @@ exports.addModel = function(database) {
 
       that.validateOnCreate()
         .then(function(post) {
-          return Promise.all([
-            database.hmsetAsync(mkKey(['post', post.id]),
-                                { 'body': post.body,
-                                  'userId': post.userId,
-                                  'createdAt': post.createdAt.toString(),
-                                  'updatedAt': post.updatedAt.toString()
-                                }),
-            models.Timeline.newPost(post.id, that.timelineIds)
```

Comment:
This was by the way a significant bug: we started a promise that tried to find a post by ID in parallel with a promise that created a database record allowing that to happen.